paging-runtime-composeui/src/commonMain/kotlin/app/cash/paging/LazyPagingItems.kt

```diff
@@ -0,0 +1,320 @@
+package app.cash.paging
+
+
+import androidx.compose.foundation.lazy.LazyItemScope
+import androidx.compose.foundation.lazy.LazyListScope
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+
+import androidx.paging.CombinedLoadStates
+import androidx.paging.DifferCallback
+import androidx.paging.ItemSnapshotList
+import androidx.paging.LOGGER
+import androidx.paging.LoadState
+import androidx.paging.LoadStates
+import androidx.paging.NullPaddedList
+import androidx.paging.PagingData
+import androidx.paging.PagingDataDiffer
+import kotlin.coroutines.CoroutineContext
+import kotlin.coroutines.EmptyCoroutineContext
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.collect // TODO("OG") Why does compiler say this is unused?
```

Comment:
Weird, `collect` is used in this file yet compiler warns me it's unused.. maybe a bug?